src/replay/execution.zig

```diff
@@ -145,7 +175,7 @@ const ReplaySlotStatus = union(enum) {
 /// - [replay_active_bank](https://github.com/anza-xyz/agave/blob/161fc1965bdb4190aa2d7e36c7c745b4661b10ed/core/src/replay_stage.rs#L2979)
 /// - [replay_blockstore_into_bank](https://github.com/anza-xyz/agave/blob/161fc1965bdb4190aa2d7e36c7c745b4661b10ed/core/src/replay_stage.rs#L2232)
 /// - [confirm_slot](https://github.com/anza-xyz/agave/blob/161fc1965bdb4190aa2d7e36c7c745b4661b10ed/ledger/src/blockstore_processor.rs#L1494)
-fn replaySlot(state: ReplayExecutionState, slot: Slot) !ReplaySlotStatus {
+fn replaySlot(state: *ReplayExecutionState, slot: Slot) !ReplaySlotStatus {
```

Comment:
Related to the other comment on ReplayExecutionState, it doesn't own anything, why is this now a mutable pointer?